src/stylesheets/elements/_buttons.scss

```diff
@@ -1,19 +1,19 @@
 .usa-button {
   @include u-margin-y("1px");
-  @include u-text('secondary-darker');
+  @include u-text("secondary-darker");
   @include u-padding-y(1.5);
   @include u-padding-x(4);
 
   &.usa-button--hover,
   &:hover {
-    @include u-bg('primary-light');
-    @include u-text('secondary-darker');
+    @include u-bg("primary-light");
+    @include u-text("secondary-darker");
   }
 
   &.usa-button--active,
   &:active {
-    @include u-bg('secondary-darker');
-    @include u-text('white');
+    @include u-bg("secondary-darker");
```

Comment:
dup

Reply:
I don't understand

Reply:
Duplicate code -- usa-button--active and --hover

Reply:
one resolves to .usa-button:active and the other to sds-button--circle:active In fact you will find this on every variant

Reply:
sds-button--circle still has the usa-button class in the njk file

Reply:
correct